commands/status.go

```diff
@@ -37,6 +38,7 @@ PMM-agent:
 	Connected : {{ .PMMAgentStatus.Connected }}{{ if .PMMAgentStatus.Connected }}
 	Time drift: {{ .PMMAgentStatus.ServerClockDrift }}
 	Latency   : {{ .PMMAgentStatus.ServerLatency }}
+	Version   : {{ .PMMVersion }}
```

Comment:
Strictly speaking, it is a version of pmm-admin but not pmm-agent.

```
[root@fa40586bdea1 /]# pmm-agent --version
ProjectName: pmm-agent
Version: 2.6.0
PMMVersion: 2.6.0
Timestamp: 2020-05-05 12:50:58 (UTC)
FullCommit: d66ebd798de738d9c1711ea02e57fb0cec5b0962
[root@fa40586bdea1 /]# pmm-admin --version
ProjectName: pmm-admin
Version: 2.5.0-7-g4ab1e51-dirty
PMMVersion: 2.5.0-7-g4ab1e51-dirty
Timestamp: 2020-05-06 09:50:59 (UTC)
FullCommit: 4ab1e51f86d9e19c0173af409bd8c7b40fa9d784
Branch: PMM-5776-inv-ex-service
```

My suggestion to add a new section:
```
PMM Admin: 
  Version   : {{ .PMMVersion }}
```

Otherwise, I guess we need to extend the Status method to return the version of pmm-agent. https://github.com/percona/pmm-agent/blob/master/agentlocal/agent_local.go#L127